scripts/LargeScaleStructures/geometry_writer.py

```diff
@@ -111,6 +112,7 @@ def addSamplePosition(self, location=None, coord_type="cartesian"):
         child = self._append_child("type", self._root, name="sample-position")
         child.setAttribute("is", "SamplePos")
 
+    # pylint:disable = dangerous-default-value
     def addDetectorPixels(self, name, r=[], theta=[], phi=[], names=[], energy=[]):
```

Comment:
Ah this warning refers to the dangers of using `[]` as default value/initializer in function arguments, which can produce extremely counterintuitive behavior: 
http://stackoverflow.com/questions/9887180/the-value-of-an-empty-list-in-function-parameter-example-here
 I wouldn't disable this warning.
It might be possible to fix it by using `None` or just not giving default values.